extras-requirements.txt

```
@@ -13,9 +13,9 @@
 s3fs>=2023.9.2, <2024.7
 gcsfs>=2023.9.2, <2024.7
 ossfs==2023.12.0
-boto3>=1.28.0,<1.29.0
+boto3>=1.35.16
```

```suggestion
boto3~=1.35.16
```
and do it across the its use. we do not give minimum only but either < or ~